app/src/main/java/org/p2p/wallet/user/interactor/TokenMetadataInteractor.kt

```diff
@@ -27,20 +27,22 @@ class TokenMetadataInteractor(
             null
         }
 
-        val lastModified = metadata?.timestamp
-        Timber.tag(TAG).i("Checking if metadata is modified since: $lastModified")
+        val ifModifiedSince = metadata?.timestamp
```

Comment:
without if please :)

Reply:
it's the header name. 
`if-modified-since`. Why should I remove it?